### PR TITLE
fix: fix up PrefetchServiceWorker with multiple Qwik containers

### PR DESCRIPTION
**Pull request by gioboa**
# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/QwikDev/qwik/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by wmertens** [comment]
LGTM, why is this draft @gioboa ?

**Comment by gioboa** [comment]
Because we are testing the fix in our POC
If it works, I will change the status

**Comment by gioboa** [comment]
The SW worker is fetching the remote bundles correctly 
image

**Review by thejackshelton — APPROVED** [review]
Amazing work 🙌

File: packages/qwik/src/server/prefetch-utils.ts [hunk 1]
```diff
@@ -23,9 +23,21 @@ export function prefetchUrlsEventScript(prefetchResources: PrefetchResource[]) {
   const data: QPrefetchData = {
     bundles: flattenPrefetchResources(prefetchResources).map((u) => u.split('/').pop()!),
   };
-  return `document.dispatchEvent(new CustomEvent("qprefetch",{detail:${JSON.stringify(data)}}))`;
+  return `(${PREFETCH_BUNDLES_CODE})(
+    document.currentScript.closest('[q\\\\:container]'),
```
Comment (Varixo) [inline packages/qwik/src/server/prefetch-utils.ts:27]: Why we need this? `[q\\:container]` is not enough?

Comment (gioboa) [reply packages/qwik/src/server/prefetch-utils.ts:27]: we need to run this in the browser to use the right one. I used the same logic we have [here](https://github.com/QwikDev/qwik/blob/646165fd5916f54839db2623bd5978efdf822a7b/packages/qwik/src/core/components/prefetch.ts#L81)

**Review by PatrickJS — APPROVED** [review]

**Pull request merged**